api/src/services/__tests__/programsService.ts

```diff
@@ -9,13 +9,17 @@ import {
   findProgramWithActivities,
   listProgramsForCurriculum,
   listProgramsWithActivities,
+  getParticipantActivityId,
+  getParticipantActivityCompletion,
+  setParticipantActivityCompletion,
 } from '../programsService';
 import {
   Program,
   ProgramActivity,
   CurriculumActivity,
   ActivityType,
   ProgramWithActivities,
+  participantActivities,
```

Comment:
when Wendy's comment in the models file is addressed, the capitalization should be addressed here as well